model/task_lifecycle.go

```diff
@@ -852,6 +852,13 @@ func MarkStart(t *task.Task, updates *StatusChanges) error {
 		}
 	}
 
+	// update task unblocked field
+	err = t.CalculateUnblockedTime()
+
```

Comment:
since this only returns an error, better to have it on one line:
`if err = t.CalculateUnblockedTime(); err != nil`

Reply:
I'm assuming `if t.UpdateUnblockedTime() != nil {` or would break?

Reply:
that is technically doable but then you couldn't use the error in the return value, which is why we use the ; oneline syntax